definitions/npm/express_v4.x.x/flow_v0.32.x-/express_v4.x.x.js

```diff
@@ -100,7 +97,6 @@ declare class express$Response extends http$ServerResponse mixins express$Reques
   type(type: string): this;
   vary(field: string): this;
   req: express$Request;
-  end: (data: string | Buffer, encoding: string) => this;
```

Comment:
@briancavalier From express's document, end comes from Node core, specifically the [response.end() method of http.ServerResponse.](https://nodejs.org/api/http.html#http_response_end_data_encoding_callback) And the type definition is [here](https://github.com/facebook/flow/blob/master/lib/node.js#L1290) and it is different from our definition. But I can't figure out how to make [this part ](https://github.com/nowait/yelp-gil-service/blob/master/src/requestHandler/logging.js#L43-L57) type check. Any thoughts?